client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -1,282 +1,285 @@
 							{
 								label: translate( 'Boost' ),
-								description: translate( 'Instant speed and SEO' ),
-								href: '/boost/',
-								icon: boostIcon,
+								tagline: translate( 'Instant speed and SEO' ),
+								href: `${ JETPACK_COM_BASE_URL }/boost/`,
 							},
 							{
 								label: translate( 'VideoPress' ),
-								description: translate( 'High-quality, ad-free video' ),
-								href: '/videopress/',
-								icon: videoIcon,
+								tagline: translate( 'High-quality, ad-free video' ),
+								href: `${ JETPACK_COM_BASE_URL }/videopress/`,
 							},
 						],
 					},
 					{
-						category: {
-							label: translate( 'Growth' ),
-							description: translate( 'Grow your audience' ),
-							href: '/features/growth/',
-							icon: growthIcon,
-						},
+						label: translate( 'Growth' ),
+						tagline: translate( 'Grow your audience' ),
+						href: `${ JETPACK_COM_BASE_URL }/features/growth/`,
 						items: [
 							{
 								label: translate( 'CRM' ),
-								description: translate( 'Connect with your people' ),
-								href: 'https://jetpackcrm.com/',
-								icon: crmIcon,
+								tagline: translate( 'Connect with your people' ),
+								href:
+									'https://jetpackcrm.com/?utm_medium=automattic_referred&utm_source=jpcom_header',
 							},
 						],
 					},
 				],
 			},
 			{
 				label: translate( 'Pricing' ),
-				href: '/pricing/',
+				href: `${ JETPACK_COM_BASE_URL }/pricing/`,
 			},
 			{
 				label: translate( 'Agencies' ),
-				href: '/for/agencies/',
+				href: `${ JETPACK_COM_BASE_URL }/for/agencies/`,
 			},
 			{
 				label: translate( 'Support' ),
-				href: '/support/',
+				href: `${ JETPACK_COM_BASE_URL }/support/`,
 			},
 			{
 				label: translate( 'Blog' ),
-				href: '/blog/',
+				href: `${ JETPACK_COM_BASE_URL }/blog/`,
 			},
 		],
 		[ translate ]
 	);
 
-	const jetpackSaleCoupon = useSelector( getJetpackSaleCoupon );
-	const [ isMenuOpen, setIsMenuOpen ] = useState( false );
-
-	const toggleMenu = () => {
-		setIsMenuOpen( ( currentState ) => ! currentState );
-	};
-	const toggleSubmenu = useCallback( ( item: HTMLElement ) => {
-		item.classList.toggle( 'is-active' );
-
-		const submenu = item.querySelector( '.jpcom-masterbar__submenu' );
-
-		if ( submenu ) {
-			submenu.classList.toggle( 'is-visible' );
-		}
-	}, [] );
-	const onSubmenuClick = useCallback(
-		( e ) => {
-			if ( window.innerWidth <= BP ) {
-				toggleSubmenu( e.currentTarget );
-			}
-		},
-		[ toggleSubmenu ]
-	);
-	const onSubmenuKeyPress = useCallback(
-		( e ) => {
-			const key = e.code || e.key;
-
-			if ( key.indexOf( 'Enter' ) > -1 && window.innerWidth <= BP ) {
-				toggleSubmenu( e.currentTarget );
-			}
-		},
-		[ toggleSubmenu ]
-	);
 	const onLinkClick = useCallback( ( e ) => {
 		recordTracksEvent( 'calypso_jetpack_nav_item_click', {
 			nav_item: e.currentTarget.getAttribute( 'href' )?.replace( JETPACK_COM_BASE_URL, '' ),
 		} );
 	}, [] );
 
+	const getLocalizedLink = useCallback(
```

Comment:
Do we not already have some shared functions to handle things like this?
Maybe [`localizeUrl`](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-utils/src/localize-url.tsx#L153) or something similar?